apps/src/music/views/TimelineSampleEvents.jsx

```diff
@@ -34,6 +35,16 @@ const TimelineSampleEvents = ({
     return currentUniqueSounds.indexOf(id);
   };
 
+  const isCurrentlyPlaying = (when, length) => {
```

Comment:
Should we check this in `TimelineElement` directly so that it works for both timeline versions?

Reply:
Good call; done.